8_omicron_analysis/nowcasts_tests.py

```diff
@@ -46,20 +41,12 @@
 end_day = '2022-01-01'
 data_end_day  = '2022-01-01' # Final date for calibration -- set this to a date before boosters started
 
-
-#start_day = '2021-05-01'
-#data_end_day = '2021-11-22'
-#end_day = '2021-12-30'
+# Set up date vectors
 date_inds = np.array([sim.day(start_day), sim.day(end_day)])
 date_len = date_inds[1]-date_inds[0]
-tvec = np.arange(date_len)
-
+tvec = sc.daterange(start_day, end_day, as_date=True)[:-1] # Remove last day
```

Comment:
Much better now to work with actual dates rather than numbers that have their labels replaced with dates at the last minute